### PR TITLE
A few small code-style improvements

### PR DESCRIPTION
**Pull request by namusyaka**

### CODE REVIEW

**Comment by b264** [comment]
Looks like sinatra could use some rubocop

**Comment by burningTyger** [comment]
not all of the changes improve readability. Writing `|k,v|` is fine imho.

File: lib/sinatra/base.rb [hunk 1]
```diff
@@ -1909,14 +1909,14 @@ class #{self.class}
...
+    set :logging, Proc.new { !test? }
     set :method_override, true
-    set :run, Proc.new { ! test? }
+    set :run, Proc.new { !test? }
     set :session_secret, Proc.new { super() unless development? }
     set :app_file, nil
 
     def self.register(*extensions, &block) #:nodoc:
-      added_methods = extensions.map {|m| m.public_instance_methods }.flatten
+      added_methods = extensions.map(&:public_instance_methods).flatten
```
Comment (davydovanton) [inline lib/sinatra/base.rb]: I think you can use `#flat_map` here 😃

Comment (namusyaka) [inline lib/sinatra/base.rb]: Right. Thanks!

**Comment by namusyaka** [comment]
@burningTyger Thanks!

**Comment by kgrz** [comment]
:+1:

**Pull request merged**